File: src/main/java/yuuki1293/panmodern/registry/Items.java
package yuuki1293.panmodern.registry;

import java.util.function.Supplier;

import net.minecraft.world.item.BlockItem;
import net.neoforged.neoforge.registries.DeferredRegister;

import yuuki1293.panmodern.PANModern;

public class Items {

    public static final DeferredRegister.Items ITEMS = DeferredRegister.createItems(PANModern.MODID);

    public static final Supplier<BlockItem> PAN_CORE_ITEM = ITEMS.registerSimpleBlockItem(Blocks.PAN_CORE_BLOCK);
}

File: src/main/kotlin/yuuki1293/panmodern/registry/Items.kt
package yuuki1293.panmodern.registry

import net.minecraft.world.item.BlockItem
import net.neoforged.neoforge.registries.DeferredRegister
import yuuki1293.panmodern.PANModern
import java.util.function.Supplier

object Items {
    val ITEMS: DeferredRegister.Items = DeferredRegister.createItems(PANModern.MODID)

    val PAN_CORE_ITEM: Supplier<BlockItem> = ITEMS.registerSimpleBlockItem(Blocks.PAN_CORE_BLOCK)
    val PAN_ADAPTER_ITEM: Supplier<BlockItem> = ITEMS.registerSimpleBlockItem(Blocks.PAN_ADAPTER_BLOCK)
}
